classes/PodsAdmin.php

```diff
@@ -1360,9 +1366,38 @@ public function admin_setup_edit_proto( $duplicate, $obj ) {
 
 		wp_localize_script( 'pods-dfv', 'podsAdminConfig', $config );
 
+		$this->admin_js_dfv_config();
+
 		pods_view( PODS_DIR . 'ui/admin/setup-edit-proto.php', compact( array_keys( get_defined_vars() ) ) );
 	}
 
+	/**
+	 * Handle setting up the JS DFV config.
+	 *
+	 * @since 2.8.0
+	 */
+	protected function admin_js_dfv_config() {
+		$config = [
+			'wp_locale'      => $GLOBALS['wp_locale'],
+			'currencies'     => PodsField_Currency::$currencies,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'currencies'" and double arrow, but found 5.

<sub>hash: 9689c9be6bd2b3f6469cd2482366a2ba</sub>